libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -4,24 +4,26 @@
       autoFocus
       matInput
       type="search"
+      [attr.aria-label]="'Enter search text to search for books'"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix
+      [attr.aria-label]="'search'">
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
+    <div class="book" data-testing="book-item" *ngFor="let b of getAllBooks$ | async">
       <div class="book--title">
         {{ b.title }}
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img [alt]="b.title" src="{{ b.coverUrl }}" />
```

Comment:
Since the cover image doesn't convey useful information, it's better to use `alt=""` so that screen readers will ignorei t.

Reply:
Addressed review comment